tests/test_data/python-sphinx.spec

```diff
@@ -67,19 +67,19 @@ Requires:       python3-Jinja2 >= 2.3
 Requires:       python3-Pygments >= 2.0
 Requires:       python3-docutils >= 0.11
 Requires:       python3-snowballstemmer >= 1.1
-Conflicts:      python3-babel = 2.0
 Requires:       python3-babel >= 1.3
-Requires:       python3-alabaster < 0.8
+Conflicts:      python3-babel = 2.0
 Requires:       python3-alabaster >= 0.7
+Requires:       python3-alabaster < 0.8
 Requires:       python3-imagesize
 Requires:       python3-requests
+Requires:       python3-sqlalchemy >= 0.9
+Requires:       python3-whoosh >= 2.0
+Requires:       python3-colorama >= 0.3.5
 Requires:       python3-nose
 Requires:       python3-mock
 Requires:       python3-simplejson
 Requires:       python3-html5lib
-Requires:       python3-sqlalchemy >= 0.9
-Requires:       python3-whoosh >= 2.0
-Requires:       python3-colorama >= 0.3.5
```

Comment:
This is not related to the fix in this pull request, but I am curious if the Requires and scripts (below) output should be deterministic. So that with every pyp2rpm run, we have the relevant changes in diff, without the same lines moving around.
@mcyprian what do you think?